windows/src/developer/TIKE/compile/CompileKeymanWeb.pas

```diff
@@ -1367,14 +1367,15 @@ procedure TCompileKeymanWeb.ReportError(line: Integer; msgcode: LongWord; const
   FCallback(line, msgcode, PWideChar(text));  // I3310
 end;
 
-function TCompileKeymanWeb.RequotedString(s: WideString): string;
+function TCompileKeymanWeb.RequotedString(s: WideString; RequoteSingleQuotes: Boolean = False): string;
 var
   i: Integer;
 begin
   i := 1;
   while i <= Length(s) do
   begin
     if (s[i] = '"') or (s[i] = '\') then begin s := Copy(s, 1, i-1)+'\'+Copy(s, i, Length(s)); Inc(i); end   // I4368
+    else if (s[i] = '''') and RequoteSingleQuotes then begin s := Copy(s, 1, i-1)+'\'+Copy(s, i, Length(s)); Inc(i); end
```

Comment:
Could this have been added to the `if` in l.1377?

Reply:
I could have done -- but made the if statement more complicated than I wanted...